src/ng/directive/input.js

```diff
@@ -1436,6 +1481,23 @@ function numberInputType(scope, element, attr, ctrl, $sniffer, $browser) {
   }
 }
 
+function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
+  numberInputType(scope, element, attr, ctrl, $sniffer, $browser);
+
+  var originalRender = ctrl.$render;
+
+  ctrl.$render = function() {
+    if (ctrl.$isEmpty(ctrl.$viewValue)) {
+      ctrl.$viewValue = '50';
```

Comment:
Sorry, I meant `(max + min) / 2` (which is equivalent to your formula).
I will correct it to avoid confusion.